packages/ethereum-plugin/src/plugin.ts

```diff
@@ -0,0 +1,23 @@
+import { createPlugin } from 'earljs/internals'
+
+import { HexStringMatcher } from './matchers/aHexString'
+import { AddressMatcher } from './matchers/anAddress'
+import { PrivateKeyMatcher } from './matchers/aPrivateKey'
+import { toBeAHexString } from './validiators/toBeAHexString'
+import { toBeAnAddress } from './validiators/toBeAnAddress'
+import { toBeAPrivateKey } from './validiators/toBeAPrivateKey'
+import { toEmit } from './validiators/toEmit'
+
+export const plugin = createPlugin({
+  matchers: {
+    aHexString: HexStringMatcher.make,
+    anAddress: AddressMatcher.make,
```

Comment:
can we add matchers against ethereum checksummed address? ie. require that address is in a checksummed format and that is valid.

https://ethereum.stackexchange.com/questions/1374/how-can-i-check-if-an-ethereum-address-is-valid

Reply:
I would also be in favor of adding a non-checksummed address matcher

Reply:
> can we add matchers against ethereum checksummed address? 

We're matching only checksummed adresses at this moment. The following code is executed in `utils.isAddress`.

```
        result = getChecksumAddress(address);

        // It is a checksummed address with a bad checksum
        if (address.match(/([A-F].*[a-f])|([a-f].*[A-F])/) && result !== address) {
            logger.throwArgumentError("bad address checksum", "address", address);
        }
```